fetch.bs

```diff
@@ -397,28 +395,28 @@ these steps:
 
  <li><p>If <var>value</var> is null, then return null.
 
- <li><p>Let <var>result</var> be the result of executing the <a>parsing structured headers</a>
- algorithm with <var ignore>input_string</var> set to <var>value</var>, and
- <var ignore>header_type</var> set to <var>type</var>.
+ <li><p>Let <var>result</var> be the result of <a>parsing structured fields</a> with
+ <var ignore>input_string</var> set to <var>value</var> and <var ignore>header_type</var> set to
+ <var>type</var>.
 
- <li><p>If parsing failed, then return failure.
+ <li><p>If parsing failed, then return null.
```

Comment:
I think it's worth an explicit note how the Fetch wrapper does not distinguish between absent and unparseable structured fields, and that this is intentional in order to encourage uniform processing across the web platform.